packages/clay-a11y/bin/clay-a11y

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+pushd . > /dev/null
+SCRIPT_PATH="${BASH_SOURCE[0]}";
+while([ -h "${SCRIPT_PATH}" ]); do
+    cd "`dirname "${SCRIPT_PATH}"`"
+    SCRIPT_PATH="$(readlink "`basename "${SCRIPT_PATH}"`")";
+done
+cd "`dirname "${SCRIPT_PATH}"`" > /dev/null
+SCRIPT_PATH="`pwd`";
+popd  > /dev/null
+
+${SCRIPT_PATH}/../node_modules/.bin/babel-node ${SCRIPT_PATH}/../src/axe-runner.js "$@"
```

Comment:
Is this windows-safe?

Reply:
I'll have to look into it. It's purpose is to provide a way to use babel-node instead of node.

Reply:
After spending a bit of time researching how easy it might be to make cross platform `npm scripts`, I'm confident to say it is a great deal of effort in this case. Furthermore, Windows 10 now supports bash scripts so there is no complete lack of support on Windows side for this script. 

Alternative solutions:
1.
- Remove any syntax that is not currently supported by `Node 8`
- Execute the a11y script with Node instead of `babel-node`
- Remove the wrapper bash script

2. 
- compile a binary with the required version of Node bundle into it (Nexe)